packages/frontend/core/src/blocksuite/blocks/ai-chat-block/types.ts

```diff
@@ -1,23 +1,24 @@
+import { CopilotMessageTag, CopilotPromptMessageRole } from '@affine/graphql';
 import { z } from 'zod';
 
 // Define the Zod schema
 const ChatMessageSchema = z.object({
   id: z.string(),
   content: z.string(),
-  role: z.union([z.literal('user'), z.literal('assistant')]),
+  role: z.nativeEnum(CopilotPromptMessageRole),
```

Comment:
should exclude 'system' in chat message

Reply:
I think it is possible not to display the system role message on the view component, but there is no need to constrain the type?

Reply:
this schema is used to validate the messages submitted by users, unrelated to whether it is displayed on the front end

Reply:
Take a look at `_deserializeHistoryChatMessages` in `chat-block-peek-view.ts`. It seems to be used to parse historical messages.

Reply:
sorry, confused the same-named schema in the server
I will review it again